recipes/grblas/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "grblas" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jim22k/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 91b05d38d8b245359ab2d8b89c066f40600ce82b5d7d2c680bf52e24e45d07d0
+
+build:
+  skip: true  # [win or py<37]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - cffi
+    - graphblas ==3.2.2
+    - pytest-runner
+  run:
+    - python
+    - cffi
+    - graphblas ==3.2.2
```

Comment:
This isn't needed as the package uses `run_exports`: https://github.com/conda-forge/graphblas-feedstock/blob/b525baeb2c4aa355a80c52c8ce1f11fc1d25691f/recipe/meta.yaml#L15
```suggestion
```